### ISSUE TITLE
Using a Vuetify Component in a "Scoped" Style

### ISSUE THREAD

**Issue by nkoterba** [vuejs/rollup-plugin-vue#119]
I'm not entirely sure whether this is a https://github.com/vuejs/rollup-plugin-vue or a https://github.com/vuetifyjs/vuetify bug or even an order of processing issue with Vue.js itself.

Long story short: I'm trying to use a component with `<style scoped>` that internally uses some of the **Vuetify** components (tab controller for example).

When the `data-v` attributes get applied to the HTML by the this plugin, they are not getting applied to the **all** the elements generated by the *vuetify* components (some do get `data-v` and some don't):

Here is the source HTML:
```html
<v-tabs dark centered v-model="active">
	<v-tabs-bar slot="activators" class="dark-toolbar">
		<v-tabs-item
				v-for="report in routes"
				:key="report.path"
				:to="report.path"
				:id="report.path"
				:router=true>
			{{report.name}}
			<!--:to="report.route"-->
		</v-tabs-item>
	</v-tabs-bar>
</v-tabs>
```

Here is the HTML output:
```html
<div data-v-57497136="" class="tabs tabs--centered tabs--scroll-bars theme--dark">
	<div data-v-57497136="" class="dark-toolbar tabs__bar">
		<div class="tabs__wrapper tabs__wrapper--scrollable">
			<ul class="tabs__container" style="transform: translateX(0px);">
				<li data-v-57497136="" class="tabs__li" id="/summary"><a href="/#/reports/summary"
				                                                         class="tabs__item--active tabs__item">Daily
					Summary</a></li>
				<li data-v-57497136="" class="tabs__li" id="/privileged-activity"><a
						href="/#/reports/privileged-activity" class="tabs__item">Privileged Activity</a></li>
				<li data-v-57497136="" class="tabs__li" id="/security-events"><a
						href="/#/reports/security-events" class="tabs__item">Security Events</a></li>
				<li data-v-57497136="" class="tabs__li" id="/network"><a href="/#/reports/network"
				                                                         class="tabs__item">Network</a></li>
			</ul>
		</div>
	</div>
</div>
```

It's clear that the `data-v` tag gets correctly applied to the first few elements, but failes to get applied to the `.tabs__wrapper`, `.tabs__container` elements as well as every `a.tabs__item` element.  

Here's a screenshot which is a bit easier to see:
![image](https://user-images.githubusercontent.com/2508944/29288682-a1f5222a-8107-11e7-8b25-975fe1713fe7.png)

**Comment by znck** [comment]
Please create a repro repository. It's hard to tell without testing.

**Comment by sunnylost** [comment]
From what I know, this is a correct behavior.

`.tabs__wrapper` was created by `vuetify`, that element doesn't know anything about `data-v-xxx` attribute, `rollup-plugin-vue` cannot manipulate that element's creation, you can check `vue-loader` in `webpack`, it should has the same problem.

**Comment by nkoterba** [comment]
@sunnylost @znck Thanks for the replies.  

What I didn't investigate was when `rollup-plugin-vue` adds the `data-v-xxx` attribute:
* during rollup/bundle time
* or adds code to the bundle so that during runtime, the attribute is added *AFTER* the all components are "rendered".

But like @sunnylost said, if the `data-v-xxx` attribute is added during bundle time, and `vuetify` adds elements, etc. during runtime rendering, there's no way for this plugin to add those missing attributes.  

Unfortunately, that makes using `scoped` styles almost useless when using nested 3rd party components.  I can't imagine I'm the only person encountering this...

Any ideas on solutions? Workarounds?  What we've resulted to is using CSS' natural cascading and selector specificity to essentially "scope" or "isolate" our styles (like we did before Vue's `<style scoped>` ability) but it seems like this should be mentioned or at least addressed in the Vue framework in general.
